lib/perl5db.t

```diff
@@ -3662,6 +3662,35 @@ EOS
     $wrapper->contents_like(qr/print "2\\n"/, "break immediately after defining problem");
 }
 
+{
+    my $wrapper = DebugWrap->new(
+        {
+            cmds =>
+            [
+                "s",
+                "s",
+                "s",
+                "s",
+                "s",
+                "s",
+                "s",
+                "q",
+            ],
+            prog => \<<'EOS',
+use v5.40.0;
+sub four ($x1, $x2, $x3, $x4) { # start sub four
+    print $x1, $x2, $x3, $x4;
+}
+four(1,2,3,4);
+EOS
```

Comment:
If I save this heredoc as a separate program, what differences should I be able to observe in the behavior of the debugger between, say, perl-5.40.0 and blead+this patch?

Reply:
If you single-step (`s`) in the debugger, you should see a lot of invisible statements being executed on the `sub four($x1, $x2, $x3, $x4) {` line; i.e. you have to repeat `s` a lot to make the debugger advance into the subroutine.

I haven't tried it, but with this patch I'd assume the debugger only stops once and immediately proceeds to the `print` in the subroutine body when you hit `s`.

Reply:
> If you single-step (`s`) in the debugger, you should see a lot of invisible statements being executed on the `sub four($x1, $x2, $x3, $x4) {` line; i.e. you have to repeat `s` a lot to make the debugger advance into the subroutine.
> 
> I haven't tried it, but with this patch I'd assume the debugger only stops once and immediately proceeds to the `print` in the subroutine body when you hit `s`.

Thanks.  In perl-5.40.0, It took me 8 `s` steps before the `print` statement executed.  In the p.r., it only took 3 `s` steps.  LGTM.

Reply:
Step through the code with "s", pre-patch this steps 6 times on the sub prolog, with patch it steps once